ihmc-robotics-toolkit/src/main/java/us/ihmc/robotics/MatrixMissingTools.java

```diff
@@ -465,11 +465,41 @@ public static void unsafe_add(DMatrixRMaj matrix, int row, int col, double value
    /**
     * Negates the input matrix in-place.
     *
-    * @param matrix the matrix to be negated.
+    * @param matrixToNegate the matrix to be negated. Modified.
     */
-   public static void negate(DMatrixRMaj matrix)
+   public static void negate(DMatrixRMaj matrixToNegate)
    {
-      CommonOps_DDRM.scale(-1.0, matrix);
```

Comment:
Efficiency, in this case. 

The scale operator performs a multiply, (a = -b * a) while the unsafe_changeSign just says (a = -a). Also, it's "unsafe" because it doesn't check that the results matrix has the same size, which is a minor speed up, but does make a big of a difference. This can result in an out of bounds exception, though, which is why we called it unsafe, similar to the standard EJML unsafe_get and unsafe_set.